bot/exts/help_channels/_channel.py

```diff
@@ -46,8 +46,19 @@ async def _close_help_post(closed_post: discord.Thread, closing_reason: _stats.C
     """Close the help post and record stats."""
     embed = discord.Embed(description=CLOSED_POST_MSG)
     embed.set_author(name=f"{POST_TITLE} closed", icon_url=CLOSED_POST_ICON_URL)
-
-    await closed_post.send(embed=embed)
+    message = ''
+
+    # Include a ping in the close message if no one else engages, to encourage them
+    # to read the guide for asking better questions
+    if closing_reason == _stats.ClosingReason.INACTIVE and closed_post.owner is not None:
+        participant_ids = {
+            message.author.id async for message in closed_post.history(limit=10, oldest_first=False)
+            if not message.author.bot
+        }
+        if participant_ids == {closed_post.owner_id}:
+            message += closed_post.owner.mention
```

Comment:
```suggestion
            message = closed_post.owner.mention
```
iadd isn't needed here since it's just the empty string anyway,